client/src/components/listofsightings.js

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect } from "react";
+import Form from "./form";
+
+function Sightings() {
```

Comment:
In general, React expects the names of your components to match the file names they are exported from. It can work without that convention, but it's good practice.